tests/conftest.py

```diff
@@ -40,3 +46,31 @@ def app():
     FlaskCLI(app)
     Babel(app)
     return app
+
+
+@pytest.fixture()
+def app_error_handler():
+    """Flask app error handler fixture."""
+    app = Flask('myapp')
+
+    # Creation of a fake theme error template file.
+    temp_dir = tempfile.mkdtemp()
+    invenio_theme_dir = os.path.join(temp_dir, 'invenio_theme')
+    os.mkdir(invenio_theme_dir)
```

Comment:
you should also clean-up (see https://pytest.org/latest/fixture.html#fixture-finalization-executing-teardown-code).